src/main/groovy/com/devsoap/vaadinflow/extensions/VaadinClientDependenciesExtension.groovy

```diff
@@ -64,9 +64,10 @@ class VaadinClientDependenciesExtension {
         compileFromSources = project.objects.property(Boolean)
         offlineCachePath = project.objects.property(String)
 
-        File mirrorFolder = Paths.get(project.rootDir.absolutePath,
+        File mirrorFolder = Paths.get(project.projectDir.absolutePath,
```

Comment:
`.gradle` folder is rootDir's folder, probably makes sense to keep `rootDir` here